ForneverMind/posts/en/2022-09-05.icfpc-2022.md

```diff
@@ -22,15 +22,15 @@ Before the contest started, we have created [a GitHub project][icfpc-2022] and a
 
 - [ForNeVeR][fornever]
 - [Minoru][minoru]
-- [portnov][]
+- [portnov][portnov]
```

Comment:
This change is not required. I tend to leave such link blocks empty, since they'll take the main block content by default. So, `[portnov][portnov]` and `[portnov][]` are identical in behavior, but the latter is shorter, so I'd prefer it. The link already works!

Reply:
Ah, I should've checked before making the change ._. I just saw that the second brackets are empty, and assumed it won't work. I dropped the commit that added this and force-pushed; you can see the diff [here](https://github.com/ForNeVeR/fornever.me/compare/7a46d0b6414ad3966a9f3127f6273ccdd498fec3..fdd5a12da22d5a005680b016a140bafb00418f62).